config/environments/production.rb.template

```diff
@@ -76,7 +76,7 @@ Autolab3::Application.configure do
   # config.active_record.auto_explain_threshold_in_seconds = 0.5
 
   # default from 
-  ActionMailer::Base.default :from => ENV.fetch('SMTP_DEFAULT_FROM','something@example.com') 
```

Comment:
I don't think config hash is defined here? config_hash is only non-nil if there exists the yml file, see line 113

Reply:
Moved it down

Reply:
I don't see this change on github?

Reply:
Should be updated now.